### PR TITLE
Conference e2e [connectivity tests]

### PR DESCRIPTION
**Pull request by shapkarin**
Also it will be nice to have a test about direct page open. Like with two tabs.

### CODE REVIEW

**Comment by shapkarin** [comment]
It's works fine if you define the right `VUE_APP_JITSI_URL`

**Comment by shapkarin** [comment]
depends on #659

File: cypress.json [hunk 1]
```diff
@@ -5,5 +5,6 @@
   "env": {
     "privateKey": "7c6e602a94f30e4ea7edabe4376314f69ba7eaa2f355ecedb339df847b6f0d80575f81ffb0a297b7725dc671da0b1769b1fc5cbe45385c7b5ad1fc2eaf1d609d",
     "publicKey": "ak_fUq2NesPXcYZ1CcqBcGC3StpdnQw3iVxMA3YSeCNAwfN4myQk"
-  }
+  },
+  "chromeWebSecurity": false
```
Comment (davidyuk) [inline cypress.json:9]: is it because of iframes?

File: tests/e2e/specs/conference.spec.js [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+  it('change route by click to the iframe "OK" button', () => {
+    const ROOM = 'SOME_ROOM_NAME';
+
+    getIframeBody().find('#enter_room_field').type(ROOM);
+
+    getIframeBody().find('#enter_room_button').click();
+    cy.window().then(({ location: { pathname } }) => {
+      expect(pathname).to.equal(`/league/${ROOM}`);
+    });
```
Comment (davidyuk) [inline tests/e2e/specs/conference.spec.js:26]:
```suggestion
    cy.location('pathname').should('eq', `/league/${ROOM}`);
```

File: tests/e2e/specs/conference.spec.js [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+const { expect } = require('chai');
+
+const getIframeDocument = () => cy
+  .get('#jitsiConferenceFrame0')
+  .its('0.contentDocument')
+  .should('exist');
+
+const getIframeBody = () => getIframeDocument()
+  .its('body').should('not.be.undefined')
+  .then(cy.wrap);
```
Comment (davidyuk) [inline tests/e2e/specs/conference.spec.js:11]: For me, it is unclear how it works

File: cypress.json (hunk 1 above)
Comment (mradkov) [reply cypress.json:9]: Jitsi requires to be running in `https` only

Comment (davidyuk) [reply cypress.json:9]: Currently, it refers to Jitsi instance at league.aeternity.org, https should work fine there.

Comment (shapkarin) [reply cypress.json:9]: it's 
```
To enable cross-domain iframe access, I will set the property chromeWebSecurity to false in the file cypress.json and re-run the test.
```
https://www.cypress.io/blog/2020/02/12/working-with-iframes-in-cypress/

File: tests/e2e/specs/conference.spec.js (hunk 3 above)
Comment (shapkarin) [reply tests/e2e/specs/conference.spec.js:9]: @davidyuk 
I can rewrite as .then(iframeBody => cy.wrap(iframeBody));

Comment (shapkarin) [reply tests/e2e/specs/conference.spec.js:9]: maybe you mean about the cy.wrap method itself?

Comment (davidyuk) [reply tests/e2e/specs/conference.spec.js:9]: yes, I can't get why do you need that at all, here is a wrap doc for reference: https://docs.cypress.io/api/commands/wrap.html#Syntax

File: tests/e2e/specs/conference.spec.js [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+    cy.visit('/league');
+  });
+
+  it('change route by click to the iframe "OK" button', () => {
+    const ROOM = 'SOME_ROOM_NAME';
+
+    getIframeBody().find('#enter_room_field').type(ROOM);
+
+    getIframeBody().find('#enter_room_button').click();
+    cy.location('pathname').should('eq', `/league/${ROOM}`);
```
Comment (davidyuk) [inline tests/e2e/specs/conference.spec.js:22]: I propose to use chaining more activally, like:
```suggestion
    cy.get('#jitsiConferenceFrame0')
     .its('0.contentDocument')
     .should('exist')
     .its('body')
     .should('not.be.undefined')
     .find('#enter_room_field')
     .type(ROOM)
     .find('#enter_room_button')
     .click()
     .location('pathname')
     .should('eq', `/league/${ROOM}`)
```

Comment (davidyuk) [reply tests/e2e/specs/conference.spec.js:22]: Also, does it test stuff inside of iframe? Isn't it enough to just ensure that iframe has a proper url? (then we will be able to enable chromeWebSecurity back)

Comment (mradkov) [reply tests/e2e/specs/conference.spec.js:22]: I think we don't need to test things inside the iframe atm.

When the integrated wallet is merged we can extend the e2e to cover connectivity tests.

Comment (shapkarin) [reply tests/e2e/specs/conference.spec.js:22]: @mradkov maybe I can create a separate branch and PR for the test propose `just ensure that iframe has a proper url` and leave this one as `connectivity tests` and future?

Comment (shapkarin) [reply tests/e2e/specs/conference.spec.js:22]: @mradkov I started it test `change iframe path by Vue route` in separate branch and will open a new PR. I think it's OK?

File: src/router.js [hunk 5]
```diff
@@ -97,8 +96,8 @@ const routes = [
     },
   },
   {
-    path: '/league',
-    name: 'league',
+    path: '/events',
+    name: 'events',
```
Comment (shapkarin) [inline src/router.js:100]: todo: leave as `league`

**Pull request merged**